environment/src/main/java/com/sequenceiq/environment/environment/flow/EnvironmentReactorFlowManager.java

```diff
@@ -34,11 +34,12 @@ public EnvironmentReactorFlowManager(EventBus eventBus, EventSender eventSender,
         this.eventFactory = eventFactory;
     }
 
-    public void triggerCreationFlow(long envId, String envName, String userCrn) {
+    public void triggerCreationFlow(long envId, String envName, String userCrn, String envCrn) {
```

Comment:
Wouldn't be easier to pass an EnvironmentDto instance to this method, instead of passing a bunch of parameters which relates to the environment? (all the fields - except userCrn - are already exists in the EnvironmentDto)
If we have to pass more environment-related values to the flow in the future, this dto could contain all of them and no need to modify this method anymore.
Just askin'.
@attilapalfi92 what do you think? not a big change, just asking for your opinion as the other reviewer.

Reply:
Makes sense